packages/permission_handler/tizen/src/type.h

```diff
@@ -1,46 +1,44 @@
 #ifndef PERMISSION_HANDLER_TYPE_H_
 #define PERMISSION_HANDLER_TYPE_H_
 
-// permission group
-enum PermissionGroup {
-  PERMISSION_GROUP_CALENDAR = 0,
-  PERMISSION_GROUP_CAMERA = 1,
-  PERMISSION_GROUP_CONTACTS = 2,
-  PERMISSION_GROUP_LOCATION = 3,
-  PERMISSION_GROUP_LOCATION_ALWAYS = 4,
-  PERMISSION_GROUP_LOCATION_WHEN_IN_USE = 5,
-  PERMISSION_GROUP_MEDIA_LIBRARY = 6,
-  PERMISSION_GROUP_MICROPHONE = 7,
-  PERMISSION_GROUP_PHONE = 8,
-  PERMISSION_GROUP_PHOTOS = 9,
-  PERMISSION_GROUP_PHOTOS_ADD_ONLY = 10,
-  PERMISSION_GROUP_REMINDERS = 11,
-  PERMISSION_GROUP_SENSORS = 12,
-  PERMISSION_GROUP_SMS = 13,
-  PERMISSION_GROUP_SPEECH = 14,
-  PERMISSION_GROUP_STORAGE = 15,
-  PERMISSION_GROUP_IGNORE_BATTERY_OPTIMIZATIONS = 16,
-  PERMISSION_GROUP_NOTIFICATION = 17,
-  PERMISSION_GROUP_ACCESS_MEDIA_LOCATION = 18,
-  PERMISSION_GROUP_ACTIVITY_RECOGNITION = 19,
-  PERMISSION_GROUP_UNKNOWN = 20,
-  PERMISSION_GROUP_BLUETOOTH = 21
+// See :
+// flutter-permission-handler/permission_handler_platform_interface/lib/src/permissions.dart
```

Comment:
How about providing a link to the file?

```suggestion
// Keep in sync with the values defined in:
// https://github.com/Baseflow/flutter-permission-handler/blob/master/permission_handler_platform_interface/lib/src/permissions.dart
```

Reply:
Thanks! I've add your suggestion as a commit

Reply:
Oops. This change has been accidentally reverted.

Reply:
OMG!!!!